rebalance/rebalance.py

```diff
@@ -490,10 +490,17 @@ def rebalanceall(plugin: Plugin, min_amount: Millisatoshi = Millisatoshi("50000s
     To be economical, it tries to fix the liquidity cheaper than it can be ruined by transaction forwards.
     It may run for a long time (hours) in the background, but can be stopped with the rebalancestop method.
     """
+    # some early checks before we start the async thread
     if plugin.mutex.locked():
         return {"message": "Rebalance is already running, this may take a while. To stop it use the cli method 'rebalancestop'."}
-    if len(get_open_channels(plugin)) <= 1:
-        return {"message": "Error: Not enough open channels to balance anything"}
+    channels = get_open_channels(plugin)
+    if len(channels) <= 1:
+        return {"message": "Error: Not enough open channels to rebalance anything"}
+    our = sum(ch["to_us_msat"] for ch in channels)
+    total = sum(ch["total_msat"] for ch in channels)
+    if total - our < Millisatoshi(min_amount) or our < Millisatoshi(min_amount):
+        return {"message": "Error: No incoming liquidity to rebalance anything"}
```

Comment:
Nit: "Error: Not enough liquidity to rebalance anything" is a better description, I think. I would not mention **incoming** liquidity, as we check both directions here.